user_guide_src/source/outgoing/localization.rst

```diff
@@ -91,6 +91,18 @@ file:
 
 .. note:: The ``useSupportedLocalesOnly()`` method can be used since v4.3.0.
 
+Setting the Locale
+==================
+
+If you want to set the locale directly, you may use
+``IncomingRequest::setLocale(string $locale)``.
+You must set supported locales in **app/Config/App.php**:
+
+.. literalinclude:: localization/003.php
+
+Any attempt to set a locale not included in this array will result in the default
+locale being set.
```

Comment:
Can you add emphasis to this? Not all users know this part.

Reply:
Make the sentence a note.

Reply:
Thanks!